### ISSUE TITLE
bug: unable to parse `extern "C" fn`

### ISSUE THREAD

**Issue by wongjiahau** [andrewbaxter/genemichaels#90]
Error:
```
Caused by:
  Error formatting file
    - file = xxx
    Caused by:
      Error doing formatting
        Caused by:
          Encountered a bug; formatted source code couldn't be re-parsed in verification step
            - line = 109
            - snippet = 0105         Library::new(&library_path)
                        0106     }.with_context(|| format!("Error opening dynamic library {:?}", library_path))?;
                        0107     let language_fn_name = format!("tree_sitter_{}", name.replace('-', "_"));
                        0108     let language = unsafe {
                        0109         let language_fn: Symbol<unsafe extern "C"fn() -> Language> =
                        0110             library
                        0111                 .get(language_fn_name.as_bytes())
                        0112                 .with_context(|| format!("Failed to load symbol {}", language_fn_name))?;
                        0113         language_fn()
                        0114     };
            - column = 51
            - err = expected `fn`
```

**Comment by andrewbaxter** [comment]
Fixed in https://github.com/andrewbaxter/genemichaels/commit/f7284297e71add1bfd55684bb87860648cba7059 released in 0.5.7 .  Missed adding a space after the `api.name` which prevented re-parse.

**Comment by andrewbaxter** [comment]
Thanks for the reports! Please do let me know if this didn't work or if you encounter any other issues!
